tutorials/resources.md

```diff
@@ -0,0 +1,158 @@
+    * `<plugin>`
+* The plugin menu on the top-right of the screen.
+
+Ignition will look for GUI plugins on the following paths, in order:
+
+1. All paths set on the `IGN_GUI_PLUGIN_PATH` environment variable
+2. [GUI plugins that are installed with Ignition Gazebo](https://github.com/ignitionrobotics/ign-gazebo/tree/master/src/gui/plugins)
+3. Other paths added by calling `ignition::gui::App()->AddPluginPath`
+4. `~/.ignition/gui/plugins`
+5. [Plugins which are installed with Ignition GUI](https://ignitionrobotics.org/api/gui/3.0/namespaceignition_1_1gui_1_1plugins.html)
+
+### Physics engines
+
+[Ignition Physics](https://ignitionrobotics.org/libs/physics)
+uses a plugin architecture and its physics engines are
+built as plugins that are loaded at run time using
+[Ignition Plugin](https://ignitionrobotics.org/libs/plugin).
+
+See the [Physics engines](physics.html)
+tutorial for more details.
+
+### Rendering engines
+
+[Ignition Rendering](https://ignitionrobotics.org/libs/rendering)
+uses a plugin architecture and its render engines are
+built as plugins that are loaded at run time using
+[Ignition Plugin](https://ignitionrobotics.org/libs/plugin).
+
+At the moment, Ignition Rendering will only look for render engine plugin
+shared libraries installed within its `<install_prefix>/lib` directory.
+Likewise, the resources used by these engines are located in Ignition
+Rendering's `<install_prefix>/share` directory.
+
+### Sensors
+
+Each unique type of sensor in
+[Ignition Sensors](https://ignitionrobotics.org/libs/sensors) is a plugin. When
+a particular sensor type is requested, the relevant plugin is loaded by
+[Ignition Plugin](https://ignitionrobotics.org/libs/plugin) and a
+sensor object is instantiated from it.
+
+At the moment, Ignition Sensors will only look for sensor plugin
+shared libraries installed within its `<install_prefix>/lib` directory.
+
+## Models, lights, actors
+
+Top-level entities such as models, lights and actors may be loaded through:
+
+* Tags in SDF world files:
+    * `<world><model>`
+    * `<world><light>`
+    * `<world><actor>`
+    * `<include><uri>` (path / URL)
+* The `/world/<world_name>/create` service:
+    * SDF file as string (`<model>` / `<light>` / `<actor>` root)
+    * Path / URL to SDF file
+    * (TODO) `ignition::msgs::Model`, `ignition::msgs::Light`
+* Within a system, using
+  [SdfEntityCreator](https://ignitionrobotics.org/api/gazebo/3.0/classignition_1_1gazebo_1_1SdfEntityCreator.html)
+  or directly creating components and entities.
+
+Ignition will look for URIs (path / URL) in the following, in order:
+
+1. All paths on the `IGN_GAZEBO_RESOURCE_PATH`\* environment variable (if
+   path is URI, scheme is stripped)
+2. Current running path / absolute path
+3. [Ignition Fuel](https://app.ignitionrobotics.org/fuel/models)
+    1. Cache (i.e. `$HOME/.ignition/fuel`)
+    2. Web server
+
+> \* The `SDF_PATH` environment variable also works in some scenarious, but
```

Comment:
`scenarious`: sounds like a serious scenario :)

there is another instance [a few lines below](https://github.com/ignitionrobotics/ign-gazebo/pull/172/files#diff-97df90ddf8e71b07502c2520534c230fR144)

Reply:
Fixed in 956354f7af767d0d5715b8be3ece83fc798535a0